server/views/pages/prisons/viewSingleSessionTemplate.njk

```diff
@@ -165,7 +166,7 @@
               text: "Location groups"
             },
             value: {
-              text: locationGroups
+              html: locationGroups
```

Comment:
@hutcheonb-moj on your last PR I think I missed these groups ones going from `html` to `text`. They need to be `html` otherwise the list markup doesn't render. I added a unit test to cover groups scenario for single session view.